api/forms.py

```diff
@@ -32,8 +32,17 @@ class CreatePropertyForm(forms.Form):
 	startPrice = forms.DecimalField(initial=600)
 	dateStart = forms.DateTimeField(initial=datetime.now())
 	dateEnd = forms.DateTimeField(initial=datetime.now())
```

Comment:
I assume `dateEnd` is the auction end date. We'll run into problems with setting it equal to now. 

You can do something like:
```
from datetime import datetime, timedelta

# This will add 10 days from today.
dateEnd = forms.DateTimeField(initial=datetime.now()+timedelta(days=10))

```